__tests__/integration.test.js

```diff
@@ -225,6 +225,22 @@ describe('PATCH: api/articles/:article_id', () => {
     })
 });
 
+describe('DELETE: api/comments/:comment_id', () => {
+    test('204: sends 204 status when comment is deleted', () => {
+        return request(app)
+        .delete('/api/comments/14')
+        .expect(204)
+    });
+    test('404: sends 404 status and message when comment id is invalid', () => {
+        return request(app)
+        .delete('/api/comments/8490')
+        .expect(404)
+        .then(({ body }) => {
+            expect(body.msg).toBe("Comment Not Found")
+        })
+    })
```

Comment:
Really good error handling test and an interesting number, but dont forget to include the test for when an article id is actually invalid. I.e. /api/comments/banana